Mathlib/Topology/IndicatorConstPointwise.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2023 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä
+-/
+import Mathlib.Algebra.IndicatorFunction
+import Mathlib.Topology.Separation
+
+/-!
+# Pointwise convergence of indicator functions
+
+In this file, we prove the equivalence of three different ways to phrase that the indicator
+functions of sets converge pointwise.
+-/
+
+
+open Filter Topology
+
+variable {α : Type*} {A : Set α}
+variable {ι : Type*} (L : Filter ι) [IsCountablyGenerated L] {As : ι → Set α}
```

Comment:
Please move some more variables here, at least `{β : Type*} [Zero β] [TopologicalSpace β]`

Reply:
Also, you don't use `[IsCountablyGenerated L]`